charts/ratify/templates/deployment.yaml

```diff
@@ -31,7 +34,11 @@ spec:
             - containerPort: 6001
           volumeMounts:
             - mountPath: "/usr/local/ratify-certs"
+              {{- if .Values.akvCertConfig.enabled }}
+              name: cert-from-akv
+              {{ else }}
               name: certs
+              {{ end }}
```

Comment:
nit: use `{{- end }}` instead of `{{ end }}` to avoid new lines

Reply:
Thank you for catching this. Fixed everywhere